boards/renesas/ek_ra8p1/Kconfig.defconfig

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2025 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_EK_RA8P1
+
+if DISPLAY
+
+if SHIELD_RTKLCDPAR1S00001BE
```

Comment:
shields should be additive on top of boards, don't think this should be here, FAO @tejlmand